cli/src/pcluster/models/cluster.py

```diff
@@ -310,6 +316,14 @@ def _validate_and_parse_config(self, suppress_validators, validation_failure_lev
 
         except ValidationError as e:
             # syntactic failure
+            # Process error message. Because there are some @pre_load(s) changing the structure of the schema,
+            # We need to modify the error message to be aligned with the structure before @pre_load(s)
+            error_massage = e.args[0]
+            if "Scheduling" in error_massage:
```

Comment:
1. let's add another method called `process_error_message`
2. We have to do the same conversion for the pre_load of the storage type.

Anyway instead of doing this explicit conversion here in the model we should keep this conversion logic in the schema layer. 
We should double check what we have in the validation error object. If we have the the entire data information (i.e. schema metadata) I'm thinking to something where the conversion logic is saved as metadata in the schema layer and here we just need to call a generic converter if defined.

Reply:
Done by moving it to ClusterSchema.
I double checked the validation error object. There is no metadata inside the object.